packages/pancake-uikit/src/widgets/WalletModal/AccountModal.tsx

```diff
@@ -14,36 +14,46 @@ interface Props {
   onDismiss?: () => void;
 }
 
-const AccountModal: React.FC<Props> = ({ account, logout, onDismiss = () => null }) => (
-  <Modal title="Your wallet" onDismiss={onDismiss}>
-    <Text
-      fontSize="20px"
-      bold
-      style={{ whiteSpace: "nowrap", overflow: "hidden", textOverflow: "ellipsis", marginBottom: "8px" }}
-    >
-      {account}
-    </Text>
-    <Flex mb="32px">
-      <LinkExternal small href={`https://bscscan.com/address/${account}`} mr="16px">
-        View on BscScan
-      </LinkExternal>
-      <CopyToClipboard toCopy={account}>Copy Address</CopyToClipboard>
-    </Flex>
-    <Flex justifyContent="center">
-      <Button
-        scale="sm"
-        variant="secondary"
-        onClick={() => {
-          logout();
-          window.localStorage.removeItem(connectorLocalStorageKey);
-          window.localStorage.removeItem(ConnectorNames.WalletConnect);
-          onDismiss();
-        }}
+const AccountModal: React.FC<Props> = ({ account, logout, onDismiss = () => null }) => {
+  const clearCache = useCallback(() => {
+    window.localStorage.removeItem(connectorLocalStorageKey);
+    if (window.localStorage.getItem(ConnectorNames.WalletConnect)) {
+      window.localStorage.removeItem(ConnectorNames.WalletConnect);
+      // Reload the webpage to disconnect walletconnect websocket.
+      window.location.reload();
```

Comment:
If we have to do this fine but I wonder if there is a cleaner solution. Can [close the web socket connection](https://developer.mozilla.org/en-US/docs/Web/API/WebSocket/close) manually (without affecting other connections)?

Reply:
I couldn't find walletconnect websocket object to close it. But i will check it again

Reply:
No luck, couldnt find anything